### ISSUE TITLE
Val acc > train acc even when finetuning dropout is set to 0

### ISSUE THREAD

**Issue by AlexTMallen** [EleutherAI/weak-to-strong#5]
For example, `python sweep.py --model_sizes=EleutherAI/pythia-410m,mistralai/Mistral-7B-v0.1 --eval_every=20 --epochs=1 --ds_name=sciq_for_lm_head_with_support --n_test_docs=1000 --max_ctx=256 --results_folder=/workspace/weak-to-strong/results`

During training of pythia-410m here, we observe these results. But this phenomenon is common.
